src/styles/blocks/month.scss

```diff
@@ -0,0 +1,52 @@
+@use "../utils/extends";
+@use "../utils/variables" as *;
+
+body {
+  padding: 0;
+  margin: 0;
+  @extend %centre-flex;
+}
+
```

Comment:
add some margin at the top so that it wont go outside screen when you hover on it
<img width="748" alt="image" src="https://user-images.githubusercontent.com/100297341/201073519-68b46409-5c21-489d-b59c-c8d9b16afea5.png">